sklearn/linear_model/tests/test_sgd.py

```diff
@@ -2144,3 +2165,21 @@ def test_loss_squared_loss_deprecated(Estimator):
         assert_allclose(est1.predict_proba(X), est2.predict_proba(X))
     else:
         assert_allclose(est1.predict(X), est2.predict(X))
+
+
+def test_quantile():
+    X, y = datasets.make_regression(
+        n_samples=1000, n_features=10, n_informative=9, random_state=1234
+    )
+    from .._quantile import QuantileRegressor
```

Comment:
Can you import it on the top of the file and you should make a full import in test file

```python
from sklearn.linear_model import QuantileRegressor
```